app/views/layouts/_main_nav.html.erb

```diff
@@ -1,6 +1,10 @@
-<% nav_links = NavigationLink.all.ordered.to_a %>
+<% default_nav_links = NavigationLink.default_section.ordered.to_a %>
+<% other_nav_links = NavigationLink.other_section.ordered.to_a %>
 
 <nav class="mb-6 <% unless user_signed_in? %>mt-4<% end %>" aria-label="Primary sidebar nav">
```

Comment:
suggested enhancement (not-blocking) - As we now have two visually distinct sets of navigation in the sidebar, it would help assistive tech users if we had two distinct nav elements too (so even if you can't see the headings, you get the benefit of the separation).

We could have the first set of links wrapped in a nav, and labelled by the `community_name` heading (via `aria-labelledby`), and the second set wrapped in a nav labelled by the "Other" heading in the same way.

It would also be more accessible if the links were wrapped in `<li>` items and contained within a `<ul>` (so screen reader users can use shortcuts to move between the list items, and hear how many are in the list). Conscious that this might be out of scope of this PR though, so happy for this to be non-blocking and we can always enhance in future 😄